### PR TITLE
ATO documentation for contributing

### PR DESCRIPTION
**Pull request by RickHawesUSDS**
This PR update `contributing.md` file with changes needed for the ATO package. It also does a DH -> ReportStream subsitution. 

## Changes
- Updated PR checklist to include explicit section about sensitive information
- Updated Contribiting.md to include explicit warning about sensitive information

## Checklist
- [ ] Tested locally?
- [ ] Ran `quickTest all`?
- [ ] Downloaded a file from http://localhost:7071/api/download
- [ ] Updated the release notes? 
- [ ] Added a test?
- [ ] Did you check for sensitive data, and remove any?
- [ ] Are additional approvals needed for this change?
- [ ] Are there potential vulnerabilities or licensing issues with any new dependencies introduced?

## Fixes 
- #issues in GitHub

## To Be Done
- Stuff still to done

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -9,14 +9,20 @@ This PR ...
 - [ ] Ran `quickTest all`?
```
Comment (jimduff-usds) [inline .github/pull_request_template.md:9]:
```suggestion
- [ ] Ran `./quick-test all`?
- [ ] Ran `./prime test --env local --run ping,end2end,merge,stracbasic` against local docker container
```

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -9,14 +9,20 @@ This PR ...
...
 - [ ] Downloaded a file from http://localhost:7071/api/download
 - [ ] Updated the release notes? 
-- [ ] Added a test?
-- [ ] Did you check for sensitive data, and remove any?
+- [ ] Added tests?
+- [ ] Did you check for sensitive data, and remove any? 
+- [ ] Does logging contain sensitive data?
```
Comment (jimduff-usds) [inline .github/pull_request_template.md:14]: PRs for code changes are a good point to add a formal security review process, but I think that's a longer checklist than you could fit here.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -78,5 +79,17 @@ Once your changes and tests are ready to submit for review:
     Note that squashing at the end of the review process should not be done. Rather, it should be done when the pull request is [integrated
     via GitHub](https://github.com/blog/2141-squash-your-commits). 
 
+### Reviewer Responsibilities
+A good code review considers many aspects of the PR
+- Will the code work and will it work in the **long-term**?
+- Is the code undertandable and readable? Is documentation needed? 
+- What are the security implications of the PR?
```
Comment (jimduff-usds) [inline CONTRIBUTING.md:86]: Security implications are the responsibility of both the developer and the reviewer.  And a checklist is needed for the things to look for.

**Review by jimduff-usds — APPROVED** [review]
I suggested we add a security review step to PRs that modify code.

File: .github/pull_request_template.md (hunk 2 above)
Comment (ronaldheft-usds) [reply .github/pull_request_template.md:14]: +1 to the log mention

**Review by ronaldheft-usds — APPROVED** [review]

**Pull request merged**